docs/guides/using-the-network-shim.md

```diff
@@ -0,0 +1,193 @@
+    "dhis2ApiVersion": "37"
+}
+```
+
+#### `./cypress.json`
+
+Interactive runs are not formally supported by the Network Shim, because `cypress open` (interactive mode) and `cypress run` have different behaviour regarding the plugin events. However, cypress is attempting to smooth things over and by setting `experimentalInteractiveRunEvents` to true it should theoretically be possible to at at least use stub mode in interactive mode, for example:
+
+```json
+{
+    "baseUrl": "APP URL",
+    "projectId": "PROJECT ID",
+    "testFiles": "TEST FILES",
+    "experimentalInteractiveRunEvents": true,
+    "video": false
+}
+```
+
+#### `./cypress/plugins/index.js`
+
+In order to execute a cypress run in a DHIS2 platform app, the `enableAutoLogin` setup is required so the user is authorised to access the app. In turn, the `enableAutoLogin` setup for users on Chrome requires the `chromeAllowXSiteCookies`. As such, it makes sense to always use this plugin alongside the `networkShim`.
+
+The `networkShim` plugin itself doesn't require the `config` object, but it does accept a second `options` argument. For most DHIS2 platform apps the default options will be correct so there will be no need to pass a second argument.
+
+```js
+const {
+    networkShim,
+    chromeAllowXSiteCookies,
+} = require('@dhis2/cypress-plugins')
+
+/*
+ * const networkShimOptions = {
+ *     hosts: [
+ *         'https://main-api.com',
+ *         'https://secondary-api.com',
+ *     ],
+ *     staticResources: [
+ *         '/endpoint-which-always-return-the-same-data',
+ *         '/another-endpoint-which-always-return-the-same-data',
+ *     ]
+ * }
+ */
+
+module.exports = (on, config) => {
+    networkShim(on /*, networkShimOptions */)
+    chromeAllowXSiteCookies(on, config)
+}
+```
+
+#### `./cypress/support/index.js`
+
+The `enableNetworkShim` setup accepts no arguments and in order to execute a cypress run in a DHIS2 platform app, the `enableAutoLogin` setup is required, which means the file should look like this:
+
+```js
+import { enableAutoLogin, enableNetworkShim } from '@dhis2/cypress-commands'
+
+enableAutoLogin()
+enableNetworkShim()
+```
+
+### Package scripts and dependencies
+
+#### `./package.json`
+
+The following `devDependencies` are required:
+
+```json
+"devDependencies": {
+    "@dhis2/cli-utils-cypress": "^#.#.#",
+    "@dhis2/cypress-commands": "^#.#.#",
+    "@dhis2/cypress-plugins": "^#.#.#",
+},
+```
+
+And it is recommended to add some convenience commands to the `scripts` section, for example:
+
+```json
+"scripts": {
+    "cy:start": "BROWSER=none yarn start",
+    "cy:open": "wait-on 'http://localhost:3000' && cypress open",
+    "cy:run": "wait-on 'http://localhost:3000' && cypress run",
+    "cy:capture": "concurrently 'yarn cy:start' 'yarn cy:run --env networkMode=capture' --kill-others --success first",
+    "cy:stub": "concurrently 'yarn cy:start' 'yarn cy:run --env networkMode=stub' --kill-others --success first",
```

Comment:
```suggestion
    "cy:capture": "concurrently 'yarn cy:run --env networkMode=capture' 'yarn cy:start' --kill-others --success first",
    "cy:stub": "concurrently 'yarn cy:run --env networkMode=stub' 'yarn cy:start' --kill-others --success first",
```

`--success first` means: The command executes successfully when the first command passed to `concurrently` exits with code 0

Reply:
I was wrong: `--success first` means that concurrently will exit with 0 or 1 based on the result of the first command that finishes, regardless of the order